stable/traefik/templates/service.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     - {{ $cidr }}
     {{- end }}
   {{- end }}
-  {{- if .Values.externalTrafficPolicy }}
+  {{- if (and .Values.externalTrafficPolicy (not (eq .Values.serviceType "ClusterIp"))) }}
```

Comment:
Is it rather "ClusterIP" than "ClusterIp" ?

By the way, I think that with DaemonSet, we do not need this block anymore ?
```
{{- if (eq .Values.deployType "Deployment") }}
type: {{ .Values.serviceType }}
  {{- if .Values.loadBalancerIP }}
  loadBalancerIP: {{ .Values.loadBalancerIP }}
  {{- end }}
  {{- if .Values.externalIP }}
  externalIPs:
    - {{ .Values.externalIP }}
  {{- end }}
  {{- if .Values.loadBalancerSourceRanges }}
  loadBalancerSourceRanges:
    {{- range $cidr := .Values.loadBalancerSourceRanges }}
    - {{ $cidr }}
    {{- end }}
  {{- end }}
  {{- if .Values.externalTrafficPolicy }}
  externalTrafficPolicy: {{ .Values.externalTrafficPolicy }}
  {{- end }}
  {{- end }}
```

Reply:
Since the chart should support both Daemonset or Deployment, the block is required. and yes, that should be ClusterIP